### PR TITLE
cxl-inline-comment-context-menu component with demo usage in theme=cxl-faq.stories.js & Greg star rating demo

### PR DESCRIPTION
**Pull request by dominik-stypula-polcode**

### CODE REVIEW

File: packages/cxl-ui/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,4401 @@
+{
```
Comment (lkraav) [inline packages/cxl-ui/package-lock.json:1]: Why is there a package-lock.json here suddenly?

Definitely use `yarn`, if you didn't, and send update with this file deleted.

File: packages/cxl-ui/src/index.js [hunk 2]
```diff
@@ -7,5 +7,6 @@ export { CXLMarketingNavElement } from './components/cxl-marketing-nav.js';
 export { CXLVaadinAccordion } from './components/cxl-vaadin-accordion.js';
 export { CXLAccordionCard } from './components/cxl-accordion-card.js';
 
+export { CxlInlineCommentContextMenu } from './components/cxl-inline-comment-context-menu';
```
Comment (lkraav) [inline packages/cxl-ui/src/index.js:10]: `Cxl` -> `CXL`

File: yarn.lock [hunk 3]
```diff
@@ -1967,6 +1976,15 @@
     rollup "^1.1.0"
     rollup-plugin-babel "^4.3.0"
 
+"@pika/plugin-build-web@^0.7.1":
```
Comment (lkraav) [inline yarn.lock:1979]: Why did this old version get installed in parallel?

**Review by lkraav — CHANGES_REQUESTED** [review]
No multi-topic PRs: please split star rating stuff ouf here an re-push this branch.

**Comment by dominik-stypula-polcode** [comment]
OK so let's close this PR, and make 2 separate PR's from my and Greg's branch.

**Pull request closed without merging**